modules/patching/patch.go

```diff
@@ -39,18 +39,15 @@ func NewPatchFromString(str string) (*Patch, error) {
 	if len(parts[0]) <= 1 {
 		return nil, errors.New("Invalid base version")
 	}
-	patch.BaseVersion, err = strconv.ParseInt(string(parts[0][1]), 10, 64)
+
+	patch.BaseVersion, err = strconv.ParseInt(string(parts[0][1:]), 10, 64)
 
 	if err != nil {
 		return nil, err
 	}
 
 	diffStrs := strings.Split(parts[1], ",\n")
 
-	if len(diffStrs) == 0 {
```

Comment:
Removed because transformation can result in a null patch, which still is valid.